packages/kbn-apm-synthtrace-client/src/lib/logs/index.ts

```diff
@@ -48,6 +54,12 @@ export type LogDocument = Fields &
   }>;
 
 class Log extends Serializable<LogDocument> {
+  constructor(fields: LogDocument, private isLogsDb: boolean) {
```

Comment:
I would propose we change the signature here.

The constructor will receive a list of Fields always and options as second argument which could be a dictionary of options where you can place `isLogsDb` as an optional param which defaults to false